package.json

```diff
@@ -39,14 +38,14 @@
     "gulp-coffeelint": "^0.6.0",
     "gulp-mocha": "^3.0.0",
     "gulp-util": "^3.0.4",
-    "in-publish": "^2.0.0",
     "isomorphic-fetch": "^2.2.1",
     "jsdoc-to-markdown": "^2.0.1",
     "karma": "^1.3.0",
     "karma-env-preprocessor": "^0.1.1",
     "mocha": "^3.0.0",
     "mochainon": "^1.0.0",
     "request": "^2.67.0",
+    "require-npm4-to-publish": "^1.0.0",
     "resin-config-karma": "^1.0.4",
     "resin-settings-client": "^3.5.2",
     "rindle": "^1.2.0",
```

Comment:
Just checked, it's used for checking if the supervisor of a device is compatible with certain features